src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -43,7 +43,25 @@
  * <pre>
  * &lt;module name=&quot;MissingSwitchDefault&quot;/&gt;
  * </pre>
- *
+ * <p>Example of violation:</p>
+ * <pre>
+ * switch (i) {
+ *  case 1:
+ *    break;
+ *  case 2:
+ *    break;
+ *  // violation
```

Comment:
violation comment should be at same location where checkstyle will flag the code in a real case. The violation wouldn't be down here but on the switch itself.

`OK` can stay where it is to show it is the addition of the default.

Reply:
Ok i changed it